apps/hash-graph/package.json

```diff
@@ -17,8 +17,8 @@
     "build:docker:prod": "docker buildx build --build-arg PROFILE=production --tag hash-graph --tag hash-graph:prod --file docker/Dockerfile ../../ --load",
     "build:docker:test": "docker buildx build --build-arg PROFILE=dev --build-arg ENABLE_TEST_SERVER=yes --tag hash-graph --tag hash-graph:test --file docker/Dockerfile ../../ --load",
     "codegen:generate-openapi-specs": "just generate-openapi-specs",
-    "dev": "HASH_GRAPH_LOG_LEVEL=info cargo run --bin hash-graph --all-features -- server",
-    "dev:test-server": "HASH_GRAPH_LOG_LEVEL=info cargo run --bin hash-graph --all-features -- test-server --api-port 4001",
+    "dev": "cargo run --bin hash-graph --all-features -- server",
+    "dev:test-server": "cargo run --bin hash-graph --all-features -- test-server --api-port 4001",
```

Comment:
Drive-by: don't force into `info`. This should be overwritten in `.env.local` if desired.